### PR TITLE
Tiny Fixes

### PR DESCRIPTION
**Pull request by ynqa**
### Overview

- Move `computeLearningRate` util.go > state.go.
  - Rename to `updateLearningRate`
- Set Word2Vec configs as private variable:
  - opt, subsampleThreshold, batchSize, and theta.
- Rename `Lower` to `ToLower` in `Config`

### CODE REVIEW

File: model/word2vec/state.go [hunk 1]
```diff
@@ -223,9 +223,9 @@ func (s *State) trainRemainderBatch(wordIDs []int, trainOne func(wordIDs []int,
 func (s *State) incrementDoneWord() {
 	for range s.cwsCh {
 		s.trainedWords++
-		if s.trainedWords%s.BatchSize == 0 {
+		if s.trainedWords%s.batchSize == 0 {
 			s.lrMutex.Lock()
-			s.currentLearningRate = computeLearningRate(s.InitLearningRate, s.Theta, s.trainedWords, s.TotalFreq())
+			s.currentLearningRate = s.computeLearningRate(s.trainedWords, s.TotalFreq())
```
Comment (chewxy) [inline model/word2vec/state.go]: if `computeLearningRate` is a method of `*State` then there is no real need to pass in `s.trainedWords` and `s.TotalFreq()`. The method can just grab the field of the `*State`.

And in that case, you should just call it `updateLearnRate`

File: model/word2vec/state.go [hunk 2]
```diff
@@ -278,9 +278,17 @@ func (s *State) endTraining() {
 	s.progress.Finish()
 }
 
+func (s *State) computeLearningRate(currentWords, totalWords int) float64 {
```
Comment (chewxy) [inline model/word2vec/state.go]: See comment above

File: model/word2vec/state.go (hunk 1 above)
Comment (ynqa) [reply model/word2vec/state.go]: Sure to fix!

**Comment by chewxy** [comment]
lgtm

**Pull request merged**